### ISSUE TITLE
Error with split module

### ISSUE THREAD

**Issue by haruiz** [MortenHoustonLudvigsen/process-promises#3]
Hi Guys, nothing critical,  I just to report that you need to include , the 'split' npm module inside the dependencies block of your module , instead of dev dependencies at package.json, because in the main file you are using it. I identified it because when I installed the module from npm allways would received the error : "split module not found"

![2017-02-28_9-42-28](https://cloud.githubusercontent.com/assets/3827551/23409814/f883fc7e-fd9a-11e6-8db2-d245a608b6bc.jpg)

**Comment by chadbr** [comment]
@haruiz did you get this working? I'm trying in a typescript 2.2 project and it's not working at all...

**Comment by haruiz** [comment]
@chadbr , yes, for me is working now, just making the change that I already reported up, show me your code, maybe I could help you

**Comment by andyfischer** [comment]
fyi this is still an issue, this module does not work on its own. I recommend switching to this other module, it works great and has a similar api https://www.npmjs.com/package/child-process-promise
